core/Pos/Binary/Class/Numbers.hs

```diff
@@ -57,121 +33,6 @@ instance NonNegative Word16
 instance NonNegative Word32
 instance NonNegative Word64
 
--- Copied from Edward Kmett's 'bytes' library (licensed under BSD3)
```

Comment:
The general sentiment here is that instead of defining ad-hoc instances for all these newtype wrappers we are piggybacking as much as we can to the `cborg` definition for these primitive types (i.e. `cborg` already gives us for free things like `encodeWord16`, `decodeWord16`, `encodeInt`, `decodeInt` etc).